site/docs/pages/identity/identity-card.mdx

```diff
@@ -35,6 +35,7 @@ import { base } from 'viem/chains';
 <IdentityCard // [!code focus] 
   address="0x4bEf0221d6F7Dd0C969fe46a4e9b339a84F52FDF" // [!code focus]
   chain={base} // [!code focus]
+  schemaId="0xf8b05c79f090979bf4a80270aba232dff11a10d9ca55c4f88de95317970f0de9"
```

Comment:
Unfortunately this is a required parameter - I'm planning to remove schemaId entirely soon so this is short term